onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalInfo.java

```diff
@@ -451,31 +252,28 @@ private String computeTimeLabel(Context context) {
         }
     }
 
-    /**
-     * Computes the arrival status label from the delay (i.e., schedule deviation), where positive
-     * means the bus is running late and negative means the bus is running ahead of schedule
-     *
-     * @param delay schedule deviation, in minutes, for this vehicle where positive
-     *              means the bus is running late and negative means the bus is running ahead of
-     *              schedule
-     * @return the arrival status label based on the deviation
-     */
-    public static String computeArrivalLabelFromDelay(Resources res, long delay) {
-        if (delay > 0) {
-            // Arriving delayed
-            return res.getQuantityString(
-                    R.plurals.stop_info_arrive_delayed, (int) delay,
-                    delay);
-        } else if (delay < 0) {
-            // Arriving early
-            delay = -delay;
-            return res
-                    .getQuantityString(
-                            R.plurals.stop_info_arrive_early,
-                            (int) delay, delay);
+    private String computeNotifyText(Context context) {
+        final String routeName = mInfo.getShortName();
+        if (mEta <= 0) {
```

Comment:
Could you actually change the order of the IF/ELSE blocks to match `computeTimeLabel()` and `computeStatusLabel()`?  I think its easier to follow the logic if we always use the same order.  I actually like your order here better as it's chronological, but given the number of other changes let's avoid refactoring `computeTimeLabel()` and `computeStatusLabel()` for now.  We can refactor all three methods later together.